e2e/stylus/BUILD.bazel

```diff
@@ -1,9 +1,21 @@
 load("@build_bazel_rules_nodejs//:defs.bzl", "nodejs_test")
-load("@npm_bazel_stylus//:index.bzl", "stylus_binary")
+load("@npm//stylus:index.bzl", "stylus")
 
-stylus_binary(
+stylus(
     name = "styles",
-    src = "test.styl",
+    srcs = ["test.styl"],
+    outs = [
+        "test.css",
+        "test.css.map",
+    ],
+    args = [
+        "test.styl",
+        "--out",
+        # Output paths must use $(location) since Bazel puts them in a platform-dependent output directory
+        "$(location test.css)",
```

Comment:
As a user, can I use a relative path here such as `../test.css`? Not that it makes sense in this example but in general is that possible?

Reply:
You linked the docs which answers the question :) https://docs.bazel.build/versions/master/be/general.html#genrule.srcs

Reply:
yeah, this will be tough for users I think, because if you just write "test.css" it will write to the input folder. if you're lucky you get an error about that folder being readonly in the sandbox, more likely it writes it and you get an error that bazel says the declared output wasn't produced.
But this is standard genrule behavior so not strictly our job to improve that?